src/utils/email.js

```diff
@@ -0,0 +1,25 @@
+import nodemailer from "nodemailer";
+import nodemailerSendgrid from "nodemailer-sendgrid";
+import dotenv from "dotenv";
+
+dotenv.config();
+const transport = nodemailer.createTransport(
+  nodemailerSendgrid({
+    apiKey: process.env.SENDGRID_API_KEY
```

Comment:
Kindly add SENDGRID_API_KEY in .env.example file .

Reply:
@geraudirad . You can check now. It is solved